.github/workflows/build.yml

```diff
@@ -85,6 +86,15 @@ jobs:
         run: .github/build-with-maven-native.sh
         if: ${{ env.AZURE_CLIENT_ID != '' && env.AZURE_TENANT_ID != '' && env.AZURE_SUBSCRIPTION_ID != '' }}
 
+      # Fix error: Client assertion is not within its valid time range.
+      - name: Az CLI login
+        uses: azure/login@v1
+        with:
+          client-id: ${{ env.AZURE_CLIENT_ID }}
+          tenant-id: ${{ env.AZURE_TENANT_ID }}
+          subscription-id: ${{ env.AZURE_SUBSCRIPTION_ID }}
+        if: ${{ env.AZURE_CLIENT_ID != '' && env.AZURE_TENANT_ID != '' && env.AZURE_SUBSCRIPTION_ID != '' }}
```

Comment:
Good catch.

Reply:
I don't quite understand this, could you explain more about this?

Reply:
@backwind1233  With the Key Vault extension, the native build takes more time and causes authentication expired. We have to login again to delete resources.